local-cli/cliEntry.js

```diff
@@ -42,7 +51,8 @@ const documentedCommands = {
   'run-android': [runAndroid, 'builds your app and starts it on a connected Android emulator or device'],
   'run-ios': [runIOS, 'builds your app and starts it on iOS simulator'],
   'upgrade': [upgrade, 'upgrade your app\'s template files to the latest version; run this after ' +
-                       'updating the react-native version in your package.json and running npm install']
+                       'updating the react-native version in your package.json and running npm install'],
+  'link': [linkShim, 'link a library'],
```

Comment:
Would it be possible to poll these docs using RNPM instead of having them configured statically?